CHANGELOG.md

```diff
@@ -2,6 +2,33 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.6.0] - 2024-12-19
+
+### 🚀 Features
+
+- Wallet integration (#371)
+- Guide user to call a chain (#316)
+- Guide user to call a contract (#306)
+- Call chain events (#372)
```

Comment:
```suggestion
- Output events after calling a chain (#372)
```

Reply:
Usually, the tool grabs the name from the commit (Call parachain events), but since we’re updating manually, I’ll use the suggested one.